### DISCUSSION TITLE
Add youtube music for provider

### DISCUSSION THREAD

**Discussion by Louis454545** [MissingCore/Music#100] [Ideas]
Add youtube music for provider like https://github.com/vfsfitvnm/ViMusic

**Comment by Maxxedison** [comment]
I up this! Idk how the legal stuff whould work but i bet they could make it work

**Comment by cyanChill** [comment]
There's currently no plans of supporting any streaming services in the app (ie: we're keeping this app strictly as a local music player). This creates issues such as cost, complexity, and legality. From looking at the ViMusic repository, they get API keys in a questionable way that raises legal concerns (and this probably won't get by Google as we're publishing on the Play Store). There's also the issue of maintainability as when YouTube changes their API format, it's going to break a lot of things (which can be seen in some of the issues for ViMusic).
